sql/item_mods.sql

```diff
@@ -3598,7 +3598,7 @@ INSERT INTO `item_mods` VALUES (11145,1,25);
 INSERT INTO `item_mods` VALUES (11145,8,8);
 INSERT INTO `item_mods` VALUES (11145,25,8);
 INSERT INTO `item_mods` VALUES (11145,386,45);
-INSERT INTO `item_mods` VALUES (11145,429,52);
+INSERT INTO `item_mods` VALUES (11145,429,38);    -- Increases Footwork's attack modifier to +38/256
```

Comment:
Can anyone tell me if the divisor here should be the 256, or 1024? I am unsure where to look...